Lib/fontTools/otlLib/builder.py

```diff
@@ -330,6 +336,17 @@ def rulesets(self):
         # Squish any empty subtables
         return [x for x in ruleset if len(x.rules) > 0]
 
+    def getCompiledSize_(self, subtables):
+        size = 0
+        for st in subtables:
+            w = OTTableWriter()
+            w["LookupType"] = CountReference(
+                {"LookupType": st.LookupType}, "LookupType"
+            )
+            copy.deepcopy(st).compile(w, self.font)
```

Comment:
i suppose the copy is necessary because the OTTableWriter modifies the subtable somehow? A comment would help clarify why you need this.